api/nakadi-event-bus-api.yaml

```diff
@@ -1558,6 +1628,20 @@ definitions:
 
   EventTypeSchema:
     properties:
+      version:
```

Comment:
1. This field is managed by Nakadi and the three digit scheme suffices.
2. Version starts on 1.0 which means it doesn't allow breaking changes (under compatibility mode it always upgrades MINOR or PATCH level, under incompatible mode it might change MAJOR).
3. Semver is widely adopted and understood by developers. It captures all the necessary semantics for versioning as far as we could map.